packages/snippet-bot/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "@google-cloud/storage": "^5.4.0",
-    "gcf-utils": "^7.1.1",
+    "gcf-utils": "^7.2.0",
```

Comment:
I just released a newer version of `gcf-utils` which should fix an issue with the last release.